app/models/repaymentHistory/RepaymentHistoryUtils.scala

```diff
@@ -34,8 +34,8 @@ object RepaymentHistoryUtils {
     }
   }
 
-  private def getMFACreditsLink(date: LocalDate, isAgent: Boolean) = {
-    val year = date.getYear
+  private def getMFAandCutoverCreditsLink(date: String, isAgent: Boolean) = {
+    val year = LocalDate.parse(date).getYear
```

Comment:
pls check up what the caller stack, we converting LocalDate to String and the parsing again to LocalDate and extracting date, its looks a bit not logical; pls review this logic